backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -59,6 +59,12 @@ trait BackendLifecycleActorFactory {
   lazy val fileHashingActorCount: Int = 50
 
   def fileHashingActorProps: Props = FileHashingActor.props(fileHashingFunction)
+  
+  /*
+   * Returns credentials that can be used to authenticate to a docker registry server
+   * in order to obtain a docker hash.
+   */
+  def dockerHashCredentials(initializationDataOption: Option[BackendInitializationData]): Seq[Any] = Seq.empty
```

Comment:
I put this in the category of "I think we *could* but I neither understand the problem well enough nor the solution space well enough to warrant bothering you about it"